[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_get_store_duplicate_messages with id: 2024-07-01_03-21-58__9c65c239-f1e6-4bf9-a87b-7632496526bc
[35mDEBUG   [0m src.steps.common:common.py:17 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.store:store.py:27 Running fixture setup: store_setup
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:17 Docker client initialized with image harbor.status.im/wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:65 WakuNode instance initialized with log path ./log/docker/publishing_node1_2024-07-01_03-21-58__9c65c239-f1e6-4bf9-a87b-7632496526bc__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:69 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:20 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:23 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:80 Generated random external IP 172.18.109.232
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:73 Generated ports ['26330', '26331', '26332', '26333', '26334']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:410 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:139 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:141 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:47 docker run -i -t -p 26330:26330 -p 26331:26331 -p 26332:26332 -p 26333:26333 -p 26334:26334 harbor.status.im/wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=26332 --rest-port=26330 --tcp-port=26331 --discv5-udp-port=26333 --rest-address=0.0.0.0 --nat=extip:172.18.109.232 --peer-exchange=true --discv5-discovery=true --cluster-id=0 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=26334 --metrics-logging=true --store=true --relay=true --nodekey=30348dd51465150e04a5d9d932c72864c8967f806cce60b5d26afeca1e77eb68
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:53 docker network connect --ip 172.18.109.232 waku ded157bace4c05ee45f2075de1be8f54f005ea319785457c15bf3a24c90556f1
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:56 Container started with ID ded157bace4c. Setting up logs at ./log/docker/publishing_node1_2024-07-01_03-21-58__9c65c239-f1e6-4bf9-a87b-7632496526bc__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:153 Started container from image harbor.status.im/wakuorg/nwaku:latest. REST: 26330
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:26330/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:256 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:26330/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.109.232/tcp/26331/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip","/ip4/172.18.109.232/tcp/26332/ws/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip"],"enrUri":"enr:-LO4QFEncwcnob6gis0u8caG8yqklCjYEreQyqXuhDDBw2JhLuMtOILHEpqDVHAI1pBBds5y_DGI5iHh5RoQrZLgQL0BgmlkgnY0gmlwhKwSbeiKbXVsdGlhZGRyc4wACgSsEm3oBmbc3QOCcnOFAAABAACJc2VjcDI1NmsxoQM3Tqpf5eFn4Jztm4gB0Y0JVSJyxyZsW8QR-QU5DZb-PYN0Y3CCZtuDdWRwgmbdhXdha3UyAw"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:261 REST service is ready !!
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:17 Docker client initialized with image harbor.status.im/wakuorg/go-waku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:65 WakuNode instance initialized with log path ./log/docker/store_node1_2024-07-01_03-21-58__9c65c239-f1e6-4bf9-a87b-7632496526bc__harbor.status.im_wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:69 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:20 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:23 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:80 Generated random external IP 172.18.65.41
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:73 Generated ports ['49526', '49527', '49528', '49529', '49530']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:410 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:139 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:141 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:47 docker run -i -t -p 49526:49526 -p 49527:49527 -p 49528:49528 -p 49529:49529 -p 49530:49530 harbor.status.im/wakuorg/go-waku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=DEBUG --rest-relay-cache-capacity=100 --websocket-port=49528 --rest-port=49526 --tcp-port=49527 --discv5-udp-port=49529 --rest-address=0.0.0.0 --nat=extip:172.18.65.41 --peer-exchange=true --discv5-discovery=true --cluster-id=0 --min-relay-peers-to-publish=1 --rest-filter-cache-capacity=50 --discv5-bootstrap-node=enr:-LO4QFEncwcnob6gis0u8caG8yqklCjYEreQyqXuhDDBw2JhLuMtOILHEpqDVHAI1pBBds5y_DGI5iHh5RoQrZLgQL0BgmlkgnY0gmlwhKwSbeiKbXVsdGlhZGRyc4wACgSsEm3oBmbc3QOCcnOFAAABAACJc2VjcDI1NmsxoQM3Tqpf5eFn4Jztm4gB0Y0JVSJyxyZsW8QR-QU5DZb-PYN0Y3CCZtuDdWRwgmbdhXdha3UyAw --storenode=/ip4/172.18.109.232/tcp/26331/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip --store=true --relay=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:53 docker network connect --ip 172.18.65.41 waku a54fc68381edcfa46900f895d9ffc8b68174a8044f8e7b35f996c9183621a67e
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:56 Container started with ID a54fc68381ed. Setting up logs at ./log/docker/store_node1_2024-07-01_03-21-58__9c65c239-f1e6-4bf9-a87b-7632496526bc__harbor.status.im_wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:153 Started container from image harbor.status.im/wakuorg/go-waku:latest. REST: 49526
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:49526/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"enrUri":"enr:-Ly4QF7nGGJ-X5kZ-eEFlFeKTA9pBBQSMBt-mCfA9p_cB_kVBMuVwDIw25lRZ9OjmIsLW0qluDRP0ikCPNWFrwI7Tg-GAZBsUEwjgmlkgnY0gmlwhKwRAAaKbXVsdGlhZGRyc5gACgSsEQAGBsF43QMACgR_AAABBsF43QOJc2VjcDI1NmsxoQLMzHlU36bou1OcgMrztAe8SHrgPDbz0tKINEtLUDllnYN0Y3CCwXeDdWRwgsF5hXdha3UyAw","listenAddresses":["/ip4/127.0.0.1/tcp/49527/p2p/16Uiu2HAm9D7zGNKp3XwH2GJomLZWE5WCeBT2N6VdgsjBXqQdrmPi","/ip4/127.0.0.1/tcp/49528/ws/p2p/16Uiu2HAm9D7zGNKp3XwH2GJomLZWE5WCeBT2N6VdgsjBXqQdrmPi","/ip4/172.17.0.6/tcp/49527/p2p/16Uiu2HAm9D7zGNKp3XwH2GJomLZWE5WCeBT2N6VdgsjBXqQdrmPi","/ip4/172.17.0.6/tcp/49528/ws/p2p/16Uiu2HAm9D7zGNKp3XwH2GJomLZWE5WCeBT2N6VdgsjBXqQdrmPi"]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:261 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:49526/admin/v1/peers" -H "Content-Type: application/json" -d '{"multiaddr": "/ip4/172.18.109.232/tcp/26331/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip", "protocols": ["/vac/waku/relay/2.0.0"], "shards": [0, 1, 2, 3, 4, 5, 6, 7, 8]}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'null'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:26330/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/0/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:49526/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/0/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'true'
[35mDEBUG   [0m src.steps.store:store.py:127 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:26330/relay/v1/messages/%2Fwaku%2F2%2Frs%2F0%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.01 seconds
[35mDEBUG   [0m src.steps.store:store.py:127 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:26330/relay/v1/messages/%2Fwaku%2F2%2Frs%2F0%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.01 seconds
[35mDEBUG   [0m src.steps.store:store.py:209 Checking that peer harbor.status.im/wakuorg/nwaku:latest can find the stored message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:26330/store/v3/messages?pubsubTopic=%2Fwaku%2F2%2Frs%2F0%2F0&pageSize=5&ascending=true" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"","statusCode":200,"statusDesc":"OK","messages":[{"messageHash":"BV3Gd51KjnYjOhlLZx/EGgWxKr8bYl3MeT5OqQVmbSQ="}]}'
[35mDEBUG   [0m src.steps.store:store.py:209 Checking that peer harbor.status.im/wakuorg/go-waku:latest can find the stored message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:49526/store/v3/messages?pubsubTopic=%2Fwaku%2F2%2Frs%2F0%2F0&contentTopics=%2Fmyapp%2F1%2Flatest%2Fproto&pageSize=5&ascending=true" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"request_id":"2b8e5de05a7f574f2f6c5259a1764a5fd66c6f3098a64a4a743e08b09592bc52","status_code":200,"status_desc":"OK","messages":[{"message_hash":"BV3Gd51KjnYjOhlLZx/EGgWxKr8bYl3MeT5OqQVmbSQ="}]}'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:200 Stopping container with id ded157bace4c
[35mDEBUG   [0m src.node.waku_node:waku_node.py:207 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:200 Stopping container with id a54fc68381ed
[35mDEBUG   [0m src.node.waku_node:waku_node.py:207 Container stopped.